scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py

```diff
@@ -89,7 +89,7 @@ def get_all_sample_ids(fq_destination_table_samples):
 def create_extract_samples_table(control_samples, fq_destination_table_samples, fq_sample_name_table, fq_sample_mapping_table):
   sql = f"CREATE OR REPLACE TABLE `{fq_destination_table_samples}` AS (" \
         f"SELECT m.sample_id, m.sample_name, m.is_loaded, m.withdrawn, m.is_control FROM `{fq_sample_name_table}` s JOIN `{fq_sample_mapping_table}` m ON (s.sample_name = m.sample_name) " \
-        f"WHERE m.is_loaded is TRUE AND m.withdrawn IS NULL AND m.is_control = {control_samples})"
+        f"WHERE m.is_loaded is TRUE AND m.is_control = {control_samples})"
```

Comment:
separate discussion, but I wonder if this encoding of "control" samples is another case where we put biz logic into GVS?

Reply:
maybe---but the biz logic is OURS for sensitivity and precision